daprdocs/content/en/operations/configuration/configuration-overview.md

```diff
@@ -97,10 +97,10 @@ The following table lists the properties for tracing:
 |--------------|--------|-------------|
 | `samplingRate` | string | Set sampling rate for tracing to be enabled or disabled.
 | `stdout` | bool | True write more verbose information to the traces
-| `otel.endpointAddress` | string | Set the Open Telemetry (OTEL) server address to send traces to
+| `otel.endpointAddress` | string | Set the Open Telemetry (OTEL) server address to send traces to. This may or may not require the https:// or http:// depending on your OTEL provider.
 | `otel.isSecure` | bool | Is the connection to the endpoint address encrypted
 | `otel.protocol` | string | Set to `http` or `grpc` protocol
-| `zipkin.endpointAddress` | string | Set the Zipkin server address to send traces to
+| `zipkin.endpointAddress` | string | Set the Zipkin server address to send traces to. This may or may not require the https:// or http:// depending on your Zipkin provider.
```

Comment:
As with above, you should include the protocol when using the `zipkin` exporter.

Reply:
Updated